enterprise/api/v1/views.py

```diff
@@ -1227,10 +1227,40 @@ def update_branding(self, request, enterprise_uuid):
 
 
 class EnterpriseCustomerCatalogWriteViewSet(EnterpriseWriteOnlyModelViewSet):
+    """
+    API write only views for the ``enterprise-customer-catalog`` API endpoint.
+    """
     queryset = models.EnterpriseCustomerCatalog.objects.all()
     permission_classes = (permissions.IsAdminUser,)
     serializer_class = serializers.EnterpriseCustomerCatalogWriteOnlySerializer
 
+    def create(self, request, *args, **kwargs):
```

Comment:
Are the request parameters declared anywhere for this view?  IMO, you should at least declare them in the docstring, if not via drf_spectacular or something more robust.

Reply:
Since we don't have drf_spectacular/redoc set up yet for this repo, honestly just listing the URL format and request body params in the docstring would be great!